ansible/roles/providers/tasks/azure-redhat.yml

```diff
@@ -18,35 +18,22 @@
 - block:
     # TODO: remove cut utility fix to initramfs after its been fixed upstream
     # https://bugzilla.redhat.com/show_bug.cgi?id=2165042
-    - name: expose cut utility to initramfs setup
+    - name: Expose cut utility to initramfs setup
       shell: echo 'install_items+=" /usr/bin/cut "' > /etc/dracut.conf.d/cut.conf
 
-    - name: refresh dracut
+    - name: Refresh dracut
       shell: dracut -f
 
-    # https://d2iq.atlassian.net/browse/D2IQ-96417 removes extra 4 and 5 partitions so that growpart cloud-init module can expand the root filesystem to the rest of the disk.
-    #
-    # [packer@pkrvmia9ljyrxtg ~]$ lsblk -oMOUNTPOINT,PKNAME -P | grep 'MOUNTPOINT="/"' | cut -d= -f3  | xargs
-    # sda
-    - name: Get Device that holds root filesystem
-      shell: lsblk -oMOUNTPOINT,PKNAME -P | grep 'MOUNTPOINT="/"' | cut -d= -f3  | xargs
-      register: rootdevice
-
-    - name: Loop over ansible block devices
-      parted:
-        number: "{{ item }}"
-        state: absent
-        device: "/dev/{{ rootdevice.stdout }}"
-      with_items:
-        - 4
-        - 5
-
-    - name: unconditionally reboot the machine with all defaults
-      ansible.builtin.reboot:
-  when:
-    - packer["image_publisher"] == "erockyenterprisesoftwarefoundationinc1653071250513" and packer["image_version"] == "9.1.20230215"
-
+    # - name: Reboot after dracut update
+    #   ansible.builtin.reboot:
+    #     msg: "Rebooting to apply initramfs changes"
+    #     pre_reboot_delay: 5
+    #     post_reboot_delay: 30
+    #     reboot_timeout: 600  # Timeout of 10 minutes to ensure full reboot
+    #     connect_timeout: 30
```

Comment:
commented out reboot task temporarily as it was causing packer build to hung up - https://github.com/mesosphere/konvoy-image-builder/actions/runs/11571038857/job/32208060620#step:8:1304

Reply:
Thanks, I removed this completely as its no longer needed.